### DISCUSSION TITLE
Performance Test

### DISCUSSION THREAD

**Discussion by zx128** [stac-utils/titiler-pgstac#87] [Show and tell]
Posted below are results of performance tests for titiler-pgstac done on a bare metal host by GIBS/VisLabs team.

![231009801-a7f12c4a-6f7f-4f4d-bdd9-beec82eeb0f9](https://user-images.githubusercontent.com/54592015/234337103-24f1174c-4637-4483-889a-3998d29a21b9.jpg)

![231009925-17ff0189-2067-480a-98b5-890a2f364e23](https://user-images.githubusercontent.com/54592015/234337685-34dc93a9-86c7-494f-8660-4a55dd949b56.jpg)

![231009982-e2374448-ad53-4dd7-bd30-3a9825782506](https://user-images.githubusercontent.com/54592015/234337755-dc71cfd4-9def-43f6-950e-f78f0a4d236e.jpg)

![231010027-11629e13-d5fb-47d0-9ef2-2aea39d2085f](https://user-images.githubusercontent.com/54592015/234338268-e1805bec-ea5d-49c1-82b9-0317ed5bf9ac.jpg)

**Comment by vincentsarago** [comment]
@zx128 as mentioned in https://github.com/developmentseed/titiler/discussions/627#discussioncomment-5577854 it would be interesting to know what environment variable are set during this tests. Without a proper reproducible benchmark it's pretty difficult to get any information from this 🤷 

I'm not sure to understand your goal of duplicating this discussion in titiler and titiler-pgstac, so please close one of them 🙏
